src/coreclr/jit/codegenarm.cpp

```diff
@@ -647,30 +647,7 @@ void CodeGen::genTableBasedSwitch(GenTree* treeNode)
 //
 void CodeGen::genJumpTable(GenTree* treeNode)
 {
-    noway_assert(compiler->compCurBB->KindIs(BBJ_SWITCH));
-    assert(treeNode->OperGet() == GT_JMPTABLE);
-
-    unsigned   jumpCount = compiler->compCurBB->GetSwitchTargets()->bbsCount;
-    FlowEdge** jumpTable = compiler->compCurBB->GetSwitchTargets()->bbsDstTab;
-    unsigned   jmpTabBase;
-
-    jmpTabBase = GetEmitter()->emitBBTableDataGenBeg(jumpCount, false);
```

Comment:
Not sure if this is important, but here you have `false `in the argument of calling

> jmpTabBase = GetEmitter()->emitBBTableDataGenBeg(jumpCount, false);

in the rest calls and new method you have `true`

> jmpTabBase = GetEmitter()->emitBBTableDataGenBeg(jumpCount, true);

why is this difference, is it needed and can it be changed?

Reply:
Yes, I think it is :) Thanks for catch.

Reply:
BTW, does this argument matter on a 32-bit platform?
https://github.com/dotnet/runtime/blob/0a5418d7e7ae7f7653a32004c808881af5275469/src/coreclr/jit/emit.cpp#L7885-L7935